modules/ROOT/taxonomy.adoc

```diff
@@ -234,17 +234,17 @@
   *** xref:admin-application-overview.adoc[Administration Applications]
    **** xref:pause-and-resume-bpm-services.adoc[Put Runtime in maintenance mode]
    **** Organization
-    ***** xref:organization-in-bonita-bpm-portal-overview.adoc[Organization Management]
+    ***** xref:organization-in-bonita-bpm-applications-overview.adoc[Organization Management]
     ***** xref:import-export-an-organization.adoc[Install/export an organization]
     ***** xref:admin-application-groups-list.adoc[Manage groups]
     ***** xref:admin-application-roles-list.adoc[Manage roles]
     ***** Manage users
      ****** xref:admin-application-users-list.adoc[Manage a user]
-     ****** xref:custom-user-information-in-bonita-bpm-portal.adoc[Custom User Information]
+     ****** xref:custom-user-information-in-bonita-applications.adoc[Custom User Information]
```

Comment:
its either "bonita-applications" or "bonita-bpm-applications". You choose but it has to be the same cause for portal it was "bpm-portal".

Reply:
Done